Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/UserDataActivity.kt

```diff
@@ -92,6 +92,8 @@ class UserDataActivity : AppCompatActivity() {
                 data!!.data!!.toString()
             )
 
+            userDataViewModel.mustRestartApp = true
```

Comment:
This should be set after the user accepts the data import warning. If the user says no to the warning, then there's no reason to restart.

Reply:
Good idea. I also figured this should be just merged with the task dialog since everything is handled there anyways.